src/components/Grid/GridComponent.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div id="grid">
+    <table
+      v-if="treeSelected!=-1"
+    >
+      <tr>
+        <th>Variable</th>
+        <th :colspan="gridAssessments.length + 1">Collection Round</th>
+      </tr>
+      <tr>
+        <th class="w-0"></th>
+        <th class="w-0"></th>
+        <th
+          v-for="assessment in gridAssessments"
+          :key="assessment.id"
+          class="text-center">
+          {{assessment.name}}
+        </th>
+      </tr>
+      <tr>
+        <th></th>
+        <td>
+          <facet-option class="selectAll">All</facet-option>
+        </td>
+        <td v-for="assessment in gridAssessments"
+            :key="assessment.id">
+          <facet-option class="selectCol"><font-awesome-icon icon="arrow-down" /></facet-option>
+        </td>
+      </tr>
+
+      <tr
+        v-for="(row, rowIndex) in grid"
+        :key="rowIndex"
+      >
+        <th>
+          <span class="title">
+            {{variables[rowIndex].label?variables[rowIndex].label:variables[rowIndex].name}}
+          </span>
+        </th>
+        <td>
+          <facet-option class="selectRow"><font-awesome-icon icon="arrow-right" /></facet-option>
+        </td>
+        <td :key="colIndex"
+            v-for="(cell,colIndex) in row">
+          <facet-option
+            @facetToggled="toggle(rowIndex, colIndex)"
+            class="selectItem"
+            :isSelected="cell.selected">
+            {{cell.count}}
+            </facet-option>
+        </td>
+      </tr>
+    </table>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+import { mapActions, mapState, mapGetters, mapMutations } from 'vuex'
+import FacetOption from '../facets/FacetOption.vue'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faArrowDown, faArrowRight, faArrowsAlt } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+library.add(faArrowDown, faArrowRight, faArrowsAlt)
+
+export default Vue.extend({
+  components: { FacetOption, FontAwesomeIcon },
+  methods: {
+    toggle (rowIndex, colIndex) {
+      this.toggleGridSelection({
```

Comment:
where is the toggleGridSelection

Reply:
is mapped to a mutation using mapMutations